client/js/admin.ts

```diff
@@ -702,6 +763,17 @@ function settingsUpdate(e: MouseEvent) {
 				}
 			}
 			branchData.confirmationBranches = allowedConfirmationBranches;
+
+			// This operation is all or nothing because it will only error if a branch was just made into an Application branch
+			try {
+				branchData.allowAnonymous = (branchRoles[i].querySelector("fieldset.applicationBranchOptions input[type=\"checkbox\"].allowAnonymous") as HTMLInputElement).checked;
```

Comment:
Can you make `fieldset.applicationBranchOptions input[type="checkbox"]` a variable or something somewhere? Seems like it's getting repeated a lot. _Or_ consider running a `querySelector()` on that element up front and `querySelector()` on the returned element for better performance and readability